src/status_im/browser/core.cljs

```diff
@@ -67,14 +67,14 @@
     (cond-> browser history-host (assoc :unsafe? (dependencies/phishing-detect history-host)))))
 
 (defn resolve-ens-content-callback [hex]
-  (let [hash (when hex (multihash/base58 (multihash/create :sha2-256 (subs hex 2))))]
+  (let [hash (when (and hex (not= hex "0x")) (multihash/base58 (multihash/create :sha2-256 (subs hex 2))))]
     (if (and hash (not= hash resolver/default-hash))
       (re-frame/dispatch [:browser.callback/resolve-ens-multihash-success constants/ipfs-proto-code hash])
       (re-frame/dispatch [:browser.callback/resolve-ens-contenthash]))))
 
 (defn resolve-ens-contenthash-callback [hex]
   (let [proto-code (when hex (subs hex 2 4))
-        hash (when hex (multihash/base58 (multihash/create :sha2-256 (subs hex 12))))]
+        hash (when hex (multihash/base58 (multihash/create :sha2-256 (subs hex 14))))]
```

Comment:
out of curiosity, what are the extra two chars? is it '0x'?

Reply:
No just data used to identify various algo used, that we ignore here.